examples/dt/neutron/3cont_2comp_1ipa-ipv4-vlan-dvr

```diff
@@ -0,0 +1,63 @@
+# Deployed Topology 3cont_2comp_1ipa-ipv4-vlan-dvr-ovn
+
+## General information
+
+| Revision | Change              |  Date      |
+| -------: | :------------------ | :-----:    |
+|     v0.1 | Initial publication | 2024-01-10 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN, VLAN                | Must have          |
+| Glance    | default                      | Must have          |
+| Cinder    | default                      | Must have          |
+| Keystone  | default                      | Must have          |
+| TLS       | default                      | Must have          |
+| Compute   | DifferentHostFilter          | Must have          |
+
+### Additional information 
+This DT uses FreeIPA
```

Comment:
why freeipa? IIRC, TLS is enabled in all osp18 jobs by default so IIUC no specific config is needed from our side for this.

Reply:
This is fixed